[DOC-306] [DOC-320] [DOC-357] [yba] 2024.1 Release management enhancement, 2024.1 version naming changes. (#22173)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/manage-deployments/upgrade-software.md

> 1. [Review major changes in previous YugabyteDB releases](../upgrade-software-prepare/). Depending on the upgrade you are planning, you may need to make changes to your automation or upgrade your Linux operating system.

Reviewer: ```suggestion
1. [Prepare to upgrade a universe](../upgrade-software-prepare/). Depending on the upgrade you are planning, you may need to make changes to your automation or upgrade your Linux operating system.
```
